application/apps/indexer/parsers/src/someip.rs

```diff
@@ -85,7 +90,12 @@ impl FibexMetadata {
             if let Some(som_type) = lock.get_som_type(fibex_type) {
                 match som_type.parse(&mut SOMParser::new(payload)) {
                     Ok(_) => {
-                        return Some(som_type.to_string().replace([' ', '\n'], ""));
+                        return Some(
+                            som_type
+                                .to_string()
+                                .replace('\n', LINE_SEP)
+                                .replace(&" ".repeat(4), "\t"),
```

Comment:
Thanks @kruss for this changes. Just one moment. If Some/IP parser already have tabs as 4 spaces in it's output, no needs to replace `\s\s\s\s` with `\t`, because on front-end (bellow in your changes) we can do just
```
this.row !== undefined
                    ? this.sanitizer.bypassSecurityTrustHtml(
                          this.row.content
                              .replace(/\u0006/gi, '<br>')
                              .replace(/\t/gi, '&nbsp;'.repeat(4))
                              .replace(/\s/gi, '&nbsp;'), // replace spaces with &nbsp; to make it visible in outout
                      )
                    : undefined;
```
I mean in current solution you can safely remove `.replace(&" ".repeat(4), "\t"),` and output in details view should stay same